src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/NotifyServiceTest.kt

```diff
@@ -0,0 +1,45 @@
+
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.service
+
+import com.nhaarman.mockitokotlin2.any
+import com.nhaarman.mockitokotlin2.anyOrNull
+import com.nhaarman.mockitokotlin2.eq
+import com.nhaarman.mockitokotlin2.isNull
+import com.nhaarman.mockitokotlin2.verify
+import com.nhaarman.mockitokotlin2.verifyZeroInteractions
+import com.nhaarman.mockitokotlin2.whenever
+import org.junit.jupiter.api.Test
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.boot.test.mock.mockito.MockBean
+import org.springframework.test.context.ActiveProfiles
+import uk.gov.service.notify.NotificationClient
+import uk.gov.service.notify.NotificationClientException
+import java.net.URI
+
+@SpringBootTest(properties = ["spring.main.lazy-initialization=true"])
+@ActiveProfiles("test", "local")
+class NotifyServiceTest {
+  @MockBean private lateinit var notificationClient: NotificationClient
+  private val url = URI.create("http://interventions.com")
```

Comment:
In case of any accidental actual call during the future life of the app -- it's usually good practice to keep these URLs fake, i.e. `https://interventions.example.com`

> The domain names example.com, example.net, example.org, and example.edu are second-level domain names in the Domain Name System of the Internet. They are reserved by the Internet Assigned Numbers Authority (IANA) at the direction of the Internet Engineering Task Force (IETF) as special-use domain names for documentation purposes.

source: https://en.wikipedia.org/wiki/Example.com